src/styles/Card.scss

```diff
@@ -0,0 +1,111 @@
+@import './constants.scss';
+
+
+$imageDimensionDesktop:300px;
+$imageDimensionMobile:425px;
+$cardMargin:20px;
+
+.card-object {
```

Comment:
card-object could maybe become card now? As it is the top level component

Reply:
Again will have style implications and so would prefer to resolve this when we have more time.